### DISCUSSION TITLE
Server Error 401 Redgifs

### DISCUSSION THREAD

**Discussion by kahlkevin1337** [Serene-Arc/bulk-downloader-for-reddit#694] [Q&A] [answered]
Hello,
if I try to download Redgif files I get a Server Error 401.
Does someone know how I can fix it?

**Comment by Serene-Arc** [comment, marked as answer]
This is a [https://github.com/aliparlakci/bulk-downloader-for-reddit/issues/679](known issue) that has been resolved but not yet released.
